src/test/components/UploadedRecordingsHome.test.js

```diff
@@ -145,4 +147,58 @@ describe('UploadedRecordingsHome', () => {
 
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  it('can rename stored profiles', async () => {
+    // Store a single profile that we will rename. The data inside is mostly
+    // arbirtrary except the "name" field.
+    // mockDate('4 Jul 2020 15:00');
+    await storeProfileData({
+      profileToken: 'FAKE_1234',
+      jwtToken: null,
+      publishedDate: new Date('4 Jul 2020 13:00'), // This is the future!
+      name: 'Initial Profile Name',
+      preset: null,
+      originHostname: 'https://mozilla.org',
+      meta: {
+        product: 'Firefox',
+        platform: 'Macintosh',
+        toolkit: 'cocoa',
+        misc: 'rv:62.0',
+        oscpu: 'Intel Mac OS X 10.12',
+      },
+      urlPath: '/public/FAKE_1234/marker-chart/',
+      publishedRange: { start: 2000, end: 40000 },
+    });
```

Comment:
If you move this test to `ListOfPublishedProfiles.test.js` you can reuse `listOfProfileInformations`, for example:
```
const profileData = {
  ...listOfProfileInformations[0],
  name: 'XXXX',
};
```

Reply:
I still think this should be moved to `ListOfPublishedProfiles.test.js` for simplicity...